### PR TITLE
feat: add setup script

### LINKED ISSUE #30: Improve Quick Start / Create Script
Right now, there is nothing "quick" about the [Quick Start](https://github.com/gregrickaby/nextjs-wordpress/blob/main/README.md#quick-start). 🤣

I would prefer there be a setup script that a user can run to stand up a development environment.

- [ ] Use the steps in "Quick Start" to create a simple script that will run on 'nix and Windows
- [ ] Put the script in the root of the project
- [ ] Add instructions to the "Quick Start" section of the README

### PR DESCRIPTION
**Pull request by leonlafa**
Closes #30

### Description

Developers can now set up their development environment by running `npm run setup` at the project's root.

### Screenshot
nextjs-wordpress-issue-30-setup-script

### Verification

How will a code reviewer test this?
[Quick start](https://github.com/gregrickaby/nextjs-wordpress#quick-start)

### CODE REVIEW

File: apps/nextjs/.env-script-test [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# WordPress URL (without trailing slash)
```
Comment (leonlafa) [inline apps/nextjs/.env-script-test]: Remove.

**Review by gregrickaby — APPROVED** [review]
@leonlafa Amazing! Thank you so much! 🍻

**Comment by leonlafa** [comment]
> @leonlafa Amazing! Thank you so much! 🍻

No worries. I wasn't able to inspect the Vercel error but I'm guessing you know what the problem is.

**Comment by gregrickaby** [comment]
I haven't fully set up Vercel yet. I need to add ENV vars, but I can't until we stand up a public backend for the frontend to query. 😅

**Pull request merged**